tests/validator/validator.cpp

```diff
@@ -946,13 +956,15 @@ TEST(Validator, integerStrings)
         "Reset in component 'component' referencing variable 'variable' has a non-integer order value ''.",
         "Reset in component 'component' referencing variable 'variable' has a non-integer order value '-'.",
         "Reset in component 'component' referencing variable 'variable' has a non-integer order value 'odd'.",
+        "Reset in component 'component' referencing variable 'variable' has an out of range integer order value '12345678901'.",
     };
```

Comment:
I thought that the integer ceiling was a whole lot higher than this ... ? Or is libcellml going to be 32-bit?

Reply:
libCellML is 64-bit, but an `int` is always 32-bit (see [here](https://en.cppreference.com/w/cpp/language/types)).

Reply:
In that case we should probably update the test for out of range prefix in the Units module to a lower number?

Reply:
We could, but it doesn't matter... or does it? The most important is that the value is indeed out of range, and it is.

Reply:
Eh, I think it does ... or at least the comment there should change.  It's my bad - I thought we were 64-bit :)

Reply:
Oops, I hadn't taken into account the comment. Agreed. Done in commit 3a810dd1.